follow/management/commands/send_stream_emails.py

```diff
@@ -71,8 +71,7 @@ def handle(self, *args, **options):
             week_first_day_str = week_first_day.strftime("%d %b").lstrip("0")
             week_last_day_str = week_last_day.strftime("%d %b").lstrip("0")
 
-            subject_str = u'new sounds from users and tags you are following ('
-            subject_str += unicode(week_first_day_str) + u' - ' + unicode(week_last_day_str) + u')'
+            extra_email_subject = unicode(week_first_day_str) + u' to ' + unicode(week_last_day_str)
```

Comment:
this no longer has (), but I guess it's a bit more difficult to do with the new format of subject line, right?

Reply:
I also thought I was also nicer without it